libs/utils/executor.py

```diff
@@ -212,12 +212,17 @@ def _setup_rootfs(self, tc):
         logging.debug('%14s - Setup RT-App run folder [%s]...',
                 'TargetSetup', self.te.run_dir)
         self.target.execute('[ -d {0} ] || mkdir {0}'\
-                .format(self.te.run_dir), as_root=True)
+                .format(self.te.run_dir))
         self.target.execute(
                 'grep schedtest /proc/mounts || '\
                 '  mount -t tmpfs -o size=1024m {} {}'\
                 .format('schedtest', self.te.run_dir),
                 as_root=True)
+        # tmpfs mounts have a context with "tmpfs" as the type (while other
```

Comment:
This concept of "context" is not really clear to me, is it referring to SELinux?
Can you please try to rephrase the issue to be clear to SELinux-idiots :-)